syft/frameworks/torch/tensors/interpreters/native.py

```diff
@@ -687,15 +705,16 @@ def move(self, location: BaseWorker, requires_grad: bool = False):
         """
         Move acts on a pointer to A to move the remote value to B (=location).
 
-        Note a A will keep a copy of his value that he sent to B. This follows the
-        .send() paradigm where the local worker keeps a copy of the value he sends.
+        Basically, this function sends data directly from one worker to other.
+        Note that A will keep a copy of its value that it sent to B. This follows the
+        .send() paradigm where the local worker keeps a copy of the value it sends.
```

Comment:
>Note that A will keep a copy of its value that it sent to B. This follows the
        .send() paradigm where the local worker keeps a copy of the value it sends.

#### Change to:
Note that A destroys its copy of the object after it is sent to B. Hence resulting in one less object in A and one extra object in B than we started with. It's behaviour differs from the `.send()` paradigm where A keeps its copy of the object it sends.

PS: This behaviour was changed in #3343

Reply:
@Syzygianinfern0 - Thanks! Do you mind adding these changes as a suggestion so that I can directly commit them?  
https://haacked.com/archive/2019/06/03/suggested-changes/